### PR TITLE
Add Toast message when message is copied

### PR DESCRIPTION
**Pull request by minervapanda**
Solves #200 Added a toast message to show that the message has been copied (enhancement)

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@minervapanda, it looks like you've already signed the Dropbox CLA. Thanks!

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -86,4 +86,5 @@
     <string name="toast_login_failed_fetching_backends">Failed to fetch Backends!</string>
     <string name="login_activity_toast_login_error">Unknown login error.</string>
     <string name="no_topic_in_message">(no topic)</string>
+    <string name="toast_text">Message Copied</string>
```
Comment (niftynei) [inline app/src/main/res/values/strings.xml]: Ideally the name of the string would more closely match it's contents.  For example, a more explicit name for this string would be "message_copied".

**Comment by niftynei** [comment]
Hi @minervapanda. i left one comment about the naming of your string, otherwise lgtm.

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (minervapanda) [inline app/src/main/res/values/strings.xml]: Sure. I am going to change this.

Comment (niftynei) [inline app/src/main/res/values/strings.xml]: Great, thanks!

**Comment by niftynei** [comment]
This has been merged, fixing #200

**Pull request closed without merging**